poses:
    first_floor:
        outside_elevator: # initial_pose
            x: 0
            y: 1.3
            Y: 1.719
        inside_elevator_first_try:
            x: -0.283
            y: 3.253
            Y: 1.57
        inside_elevator_second_try: # same as - /exit_elevator.yaml poses: first_floor: inside_elevator:
            x: -0.15 
            y: 3.3
            Y: 0
    second_floor:
        outside_elevator: # initial_pose
            x: 5.800
            y: 4.353
            Y: 1.907
        inside_elevator_first_try:
            x: 5.39
            y: 6.876
            Y: 1.57
        inside_elevator_second_try: # same as - /exit_elevator.yaml poses: second_floor: inside_elevator:
            x: 5.657
            y: 6.712
            Y: -0.343
